### ISSUE TITLE
Clear indication of Firewall / NAT required for VPS

### ISSUE THREAD

**Issue by itsgps** [Ysurac/openmptcprouter-vps#100]
I am seeing in multiple places that the install script sets all the relevant Ports for services, and Firewalls rules and such.  But no-where do I see it clear as to which Ports I should be NATing in to the VPS Server - which sits behind a Firewall in the DMZ area of my network.

i.e. do I forward ALL TCP and UDP ports indicated ? or is it just 65301 for instance...or 65001 and 65301... ?!

**Comment by Ysurac** [comment]
Remember that VPS should have a connection with higher bandwidth than all your aggregated connections. It's why it should be in a datacenter.
For port you need to forward rande 65000-65535 or all indicated TCP/UDP ports.

**Comment by itsgps** [comment]
Yeah no problems there - the VPS is on my own infrastructure, DELL R6xxx Servers, 10GbE Fibre NAS, et all. and on FTTP 250mb/250mb

I have forwarded the whole range for TCP/UDP to the VPS on DMZ - and tested with an ESXi OMR on a QOTOM 4 port MiniPC.  Seems like it worked perfectly I would say...As soon as I did a Speed test from a Laptop plugged into the OMR - speedtest.net was showing the Origin IP as being my own Fibre IP!  I am reasonably sure too that without any further configuration it seemed to be higher uploads with both Nighthawk routers connected...(The Download itself was limited by the VPS Fibre link which is understandable...)

Many thanks for your hard work on OMR - so far loving it!  Next I just need to work out a few things like being able to send directly to the RTMP servers on the same DMZ i.e. the have the VPS not have to trombone back out of the Site Firewall just to come back in...it's part of a greater plan to use the OMR for Streaming RTMP traffic to our OSP (OpenStreamingPlatform server, which then distributes the RTMP to various end points like Youtube and Facebook, etc.))

I'll look through some other posts to see if anything further shows up, as well as going through more of the setup/tuning documentation - since so far I was just sort of stuck on how much NATing I needed to do.

Regards
